Tribler/Test/Core/Modules/RestApi/test_create_torrent_endpoint.py

```diff
@@ -0,0 +1,74 @@
+import json
+import base64
+import os
+
+from Tribler.Core.Utilities.twisted_thread import deferred
+from Tribler.Core.TorrentDef import TorrentDef
+from Tribler.Test.Core.Modules.RestApi.base_api_test import AbstractApiTest
+from Tribler.Test.test_as_server import TESTS_DATA_DIR
+
+
+class TestMyChannelCreateTorrentEndpoint(AbstractApiTest):
+
+    def tearDown(self):
+        super(TestMyChannelCreateTorrentEndpoint, self).tearDown()
+        torrent_path = os.path.join(TESTS_DATA_DIR, os.path.basename(TESTS_DATA_DIR) + ".torrent")
+        if os.path.exists(torrent_path):
```

Comment:
Why are you removing data from the tests directory? The tests directory should only contain static data. If you need to manipulate files, please copy them to the created temporary directory first (`self.session_base_dir` iirc).